README.md

```diff
@@ -155,7 +155,8 @@ The special value `base` (the default) can be used to add the descriptor to the
 Putting the descriptor under `META-INF/versions` can help to increase compatibility with older libraries scanning class files that may fail when encountering the `module-info.class` file
 (as chances are lower that such tool will look for class files under `META-INF/versions/...`).
 
-The `jdepsExtraArgs` option can be used to specify a list of arguments passed to the _jdeps_ invocation for creating a "candidate descriptor".
+The existing JAR will be modularized and a copy of this modularized JAR can be found in the folder given via `outputDirectory`. If `overwriteExistingFiles` is `true`, existing files in the output directory will be overwritten; if `false` (the default) the task will fail if the output file already exists.
```

Comment:
I am not completely sure how useful the `outputDirectory` actually is for `addMainModuleInfo` since the original JAR is modified anyways. The current behavior also forces one to set `overwriteExistingFiles = true` to avoid errors when not performing a `gradlew clean`, see also https://github.com/moditect/moditect/issues/161.